tests/unit/migrations/bigint_nic.sql

```diff
@@ -0,0 +1,1021 @@
+-- Dump taken just after migrating nic to BIGINT primary keyA
```

Comment:
1. Just putting a file here doesn't do anything -- if you want it to be used in a test, you have to add a test that uses it (though can just be an entry in the list in migrations.py).
2. It's not clear what this is useful for testing? It looks like it's just an empty database.